contracts/Boba_SpokePool.sol

```diff
@@ -19,12 +26,6 @@ contract Boba_SpokePool is Ovm_SpokePool {
         address _crossDomainAdmin,
         address _hubPool
     ) public initializer {
-        __OvmSpokePool_init(
-            _initialDepositId,
-            _crossDomainAdmin,
-            _hubPool,
-            0x4200000000000000000000000000000000000006,
-            0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000
-        );
+        __OvmSpokePool_init(_initialDepositId, _crossDomainAdmin, _hubPool, 0x4200000000000000000000000000000000000006);
```

Comment:
This is academic since we don't support Boba, but was there a reason not to use `Lib_PreeployAddresses.OVM_ETH` instead of the hardcoded `0x42...06` address?